x/datapool/keeper/pool_distribute.go

```diff
@@ -0,0 +1,226 @@
+package keeper
```

Comment:
It would be better to rename this file to `pool_distribution.go` or `pool_dist.go` (as a noun).
Also, please rename all other `*Distribute`s to `*Distribution`.

Reply:
Thank you. I have changed to your recommendation.
https://github.com/medibloc/panacea-core/pull/318/commits/b398ff659c3333c09e78ccf4def88f4dd20dc9f2